config/devel/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+bases:
+- ../default
+
+commonLabels:
```

Comment:
I'm not exactly sure how kustomize works but if it uses the base of `../default` do we still need to define commonLabels in here? Because it looks like they're also defined there as well

Reply:
You're right, I probably included it while testing something and forgot to remove. Thanks!